app/helpers/application_helper.rb

```diff
@@ -276,7 +276,18 @@ def number_to_ssn(number)
   def number_to_obscured_ssn(number)
     return unless number
     number_to_ssn(number)
-    number.to_s.gsub!(/\w{3}-\w{2}/, '***-**')
+    if EnrollRegistry.feature_enabled?(:mask_ssn_ui_fields)
```

Comment:
rather than doing the number gsub, you should just be able to return the circle string